agent/config/builder.go

```diff
@@ -1004,16 +1006,26 @@ func (b *builder) build() (rt RuntimeConfig, err error) {
 		EnableRemoteScriptChecks:   enableRemoteScriptChecks,
 		EnableLocalScriptChecks:    enableLocalScriptChecks,
 		EncryptKey:                 stringVal(c.EncryptKey),
-		EncryptVerifyIncoming:      boolVal(c.EncryptVerifyIncoming),
-		EncryptVerifyOutgoing:      boolVal(c.EncryptVerifyOutgoing),
-		GRPCPort:                   grpcPort,
-		GRPCAddrs:                  grpcAddrs,
-		HTTPMaxConnsPerClient:      intVal(c.Limits.HTTPMaxConnsPerClient),
-		HTTPSHandshakeTimeout:      b.durationVal("limits.https_handshake_timeout", c.Limits.HTTPSHandshakeTimeout),
-		KeyFile:                    stringVal(c.KeyFile),
-		KVMaxValueSize:             uint64Val(c.Limits.KVMaxValueSize),
-		LeaveDrainTime:             b.durationVal("performance.leave_drain_time", c.Performance.LeaveDrainTime),
-		LeaveOnTerm:                leaveOnTerm,
+		StaticRuntimeConfig: StaticRuntimeConfig{
+			EncryptVerifyIncoming: boolVal(c.EncryptVerifyIncoming),
+			VerifyIncoming:        boolVal(c.VerifyIncoming),
+			VerifyIncomingHTTPS:   boolVal(c.VerifyIncomingHTTPS),
+			VerifyIncomingRPC:     boolVal(c.VerifyIncomingRPC),
+			VerifyOutgoing:        verifyOutgoing,
+			VerifyServerHostname:  verifyServerName,
+			EncryptVerifyOutgoing: boolVal(c.EncryptVerifyOutgoing),
+			CAFile:                stringVal(c.CAFile),
+			CAPath:                stringVal(c.CAPath),
```

Comment:
Can you sort these please? I think generally the RuntimeConfig has sorted fields since there's soooooo many it helps in locating what you want to change.